hummingbot/strategy/amm_arb/amm_arb_config_map.py

```diff
@@ -62,7 +62,7 @@ def order_amount_prompt() -> str:
         default="amm_arb"),
     "connector_1": ConfigVar(
         key="connector_1",
-        prompt="Enter your first spot connector (Exchange/AMM) >>> ",
```

Comment:
We do not need to make any change in this file.

Reply:
Having "spot" in the prompt is the condition in `_complete_spot_connectors()`. Having "spot" in the prompt will make it s.t. only `_spot_completer` would be used. 
Also, I think using "spot" is not accurate in this case, because the valid connectors include spot exchanges and amm exchanges.